index.html

```diff
@@ -1933,6 +1924,20 @@ <h3>Verification Material</h3>
 an example of a public key with a compound key identifier.
             </p>
           </dd>
+          <dt><dfn>publicKeyMultibase</dfn></dt>
+          <dd>
+            <p>
+The <code>publicKeyMultibase</code> property is OPTIONAL. This feature is
+non-normative. If present, the value MUST be a <a
+data-cite="INFRA#string">string</a> representation of a [[?MULTIBASE]] encoded
+public key.
+            </p>
+            <p class="advisement">
+Representations of <code>privateKeyMultibase</code> might not 
+be possible in some cases where <code>publicKeyMultibase</code> can be
+represented.
```

Comment:
It took me several readings to understand this note. I think my rephrasing and added bolding will help.
```suggestion
There might be some cases where <code><b>public</b>KeyMultibase</code> can
be represented although <code><b>private</b>KeyMultibase</code> cannot.
```

Reply:
@TallTed thank you!